docs/examples/pooling_utilities.md

```diff
@@ -0,0 +1,96 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+mystnb:
+  execution_timeout: 100
+---
+# Pooling Utility
+
+Psignifit comes with an automatic pooling utility, which can pool trials of very similar stimulus levels into blocks to reduce the time required for fitting, to profit from the beta-binomial model and to improve interpretability of data-plots.
+
+In contrast to the matlab implementation, python-psignifit does not pool implicitly. Instead a warning is printed, if pooling might be useful. Then pooling can be run as a separate call using psignifit.tools.pool_blocks
+
+This way of pooling data is presumably acceptable for many datasets, but pooling by hand is probably more exact.
+
+```{code-cell} ipython3
+
+#act = pool_blocks(inp, max_tol=0.4, max_gap=10, max_length=10)
```

Comment:
This line is commented out, is it meant to show how the utility is called? Maybe introducing it with "the general form of the manual pooling utility looks like this:" might make sense

Reply:
done